<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::StateMachineViewer</class>
 <widget class="QWidget" name="GammaRay::StateMachineViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>594</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="verticalSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>State machine:</string>
            </property>
            <property name="buddy">
             <cstring>stateMachinesView</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="stateMachinesView">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="singleStateMachineView">
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <attribute name="headerStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="maximumBlockCount">
        <number>100</number>
       </property>
       <property name="centerOnScroll">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionStartStopStateMachine">
   <property name="text">
    <string>Start/Stop State Machine</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
